src/get_target_data.py

```diff
@@ -357,7 +365,17 @@ def write_target_data(
     time_series.collect().write_parquet(ts_output_path)
     logger.info(f"Target time series saved to {ts_output_path}")
 
-    return (ts_output_path, Path())
+    # write oracle output data
+    target_oracle_output_dir = target_data_dir / "oracle-output"
+    oracle_output_path = target_oracle_output_dir / f"nowcast_date={nowcast_string}"
+    oracle_output_path.mkdir(exist_ok=True, parents=True)
+    oracle_output_path = oracle_output_path / "oracle.parquet"
```

Comment:
I realized that I have a couple of questions about hive partitioning:

1. Is it required to have `n` at the end of the file name, e.g. `"oracle-0.parquet"`?  I'm under the impression that this is necessary, but I wasn't able to find an authoritative source after a few minutes of digging.  If so, we'd also want to change the `timeseries` file name above.
2. If we're partitioning on the `nowcast_date`, do we also need it to be in the data frame itself (as in line 336 above), or is that rendered unnecessary?  I'm not sure what standard/recommended practice is.  It seems like it's helpful to have it in there so that a direct `read_parquet` gets all the columns we might want...

Reply:
i guess, looking at this, i wonder if we should also be adding the nowcast date into the timeseries file.

Reply:
@elray1 This is actually a great example of a LDR candidate. When we decide
- on the final filename conversion for oracle and time series target files
- whether or not to add nowcast_date as a column

I'll submit an LDR PR with our reasons (the latter point, especially, was a bit contentious when we debated it for the model-output files on S3)

Reply:
Your recommendations for both questions make sense to me.  For the second, I'd want to be sure that if we add the column in, standard hive-aware tools don't run into any issues.  It sounds like you have already checked into that?  If that's all fine, including the column makes sense to me.

I think it would make sense to tie these in to the hubverse discussion about formats/where/how these data are recommended to be stored in hubs.  starting with an LDR for what we've done in the variant nowcast hub sounds good, but/and i think your knowledge about these things should also inform the general hubverse recommendations :)

Reply:
so i guess the summary for purposes of this PR is that what's here looks good, but maybe we should go back and also add the nowcast date column into the time series data?

Reply:
Sounds good--do you think we should also add a `sequence_as_of` column to the time series file?

Reply:
yeah, that makes sense/would be consistent